### PR TITLE
Default value table + reponse term handling in Forms

### PR DESCRIPTION
**Pull request by sebastiankb**
[The default value table](https://w3c.github.io/wot-thing-description/#sec-default-values) provides an empty cells for contentType, safe and idempotent. This PR assign the corresponding class + explains that the response term has to be used when the response content type differ from the form content type.

### CODE REVIEW

**Comment by sebastiankb** [comment]
also see https://github.com/w3c/wot-thing-description/issues/611

File: index.html [hunk 1]
```diff
@@ -1668,111 +1679,110 @@ <h3>Default Value Definitions</h3>
...
+                <td><code>ActionAffordance</code></td>
+                <td>
+                    <code>idempotent</code>
+                </td>
+                <td><code>false</code></td>
+            </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-op-properties">
+                <td><code>PropertyAffordance</code></td>
+                <td>
+                    <code>op</code>
```
Comment (mkovatsc) [inline index.html]: Does not work for "Class". Did work when it was context, as op is not part of *Affordance...
Seems to be a big with the change from context.

Comment (sebastiankb) [reply index.html]: agree. should be changed back to Context

**Comment by sebastiankb** [comment]
thanks @mkovatsc for your comment. new commit is pushed

**Comment by sebastiankb** [comment]
as agreed with @vcharpenay Im going to merge the PR. Eventually @vcharpenay does some further updates on the table for default values

**Pull request merged**